assets/js/admin/lesson-edit.js

```diff
@@ -33,4 +31,33 @@ domReady( () => {
 	if ( moduleOptionElements.length > 0 ) {
 		moduleOptionElements.select2( { width: 'resolve' } );
 	}
+
+	// Refresh the prerequisite meta box when to course changes in order to get the relevent prerequisites.
```

Comment:
Minor typos here: 

Refresh the prerequisite meta box when ~to~ **the** course changes in order to get the ~relevent~ **relevant** prerequisites.